lib/src/base-styles/_nx-lists.scss

```diff
@@ -301,3 +301,61 @@
   margin-left: var(--nx-spacing-6x);
   max-width: var(--nx-width-text-wrapping);
 }
+
+.nx-list--numbered {
```

Comment:
Surely a lot of this duplicated the bulleted list styling? I'd want to see the code duplication minimized, probably by just using a comma selector (`.nx-list--numbered, .nx-list--bulleted`)

Reply:
> Surely a lot of this duplicated the bulleted list styling? I'd want to see the code duplication minimized, probably by just using a comma selector (`.nx-list--numbered, .nx-list--bulleted`)

Sure, consolidated here: https://github.com/sonatype/sonatype-react-shared-components/pull/616/commits/28945332181e9d2008eaf4f22fdd67c046d4ff66